common/app/templates/inlineJS/nonBlocking/atomsJs.scala.js

```diff
@@ -0,0 +1,17 @@
+@import play.twirl.api.HtmlFormat
+@(page: model.Page)(implicit context: model.ApplicationContext)
+(function () {
+  function loadAtoms() {
+    @* 
+     * each atomsJS element looks like this:
+     *   this.profile=(function(...){...}())
+     * we can just bind this to an arbitrary object that we can pass around.   
+     *@
+    @page.metadata.atomsJS.mkString(" ")
```

Comment:
For some reason, the javascript is escaped (quotes are backslashed), but even wrapping in `HtmlFormat.raw` does not change anything 🤔 😲